modules/querying-kubelet-status-on-a-node.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * support/verifying-node-health.adoc
+
+[id="querying-kubelet-status-on-a-node_{context}"]
+= Querying kubelet status on a node
+
+Review cluster node health status, resource consumption statistics and node logs. Additionally, query `kubelet` status on individual nodes.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. kubelet is managed using a systemd service on each node. Review kubelet status by querying the `kubelet` systemd service within a debug Pod:
+.. Start a debug Pod for a node:
++
+----
+$ oc debug node/my-node
+----
++
+.. Set /host as the root directory within the debug shell. The debug Pod mounts the node's root filesystem in `/host` within the Pod. By changing the root directory to `/host`, we can run binaries contained in the node's executable paths:
```

Comment:
The first /host should probably be in backticks too

Reply:
Resolved throughout the PR for `/host` references.